batch/requirements.txt

```diff
@@ -1,3 +1,3 @@
-black==22.12.0
```

Comment:
I don't see any usage of `black` in batch, so we should be safe to remove this requirement.